scrollingelement.js

```diff
@@ -28,9 +28,27 @@ if (!('scrollingElement' in document)) {
 			return isFrameset ? null : body;
 		};
 
-		Object.defineProperty(document, 'scrollingElement', {
-			'get': scrollingElement
-		});
+		if ('defineProperty' in Object) {
+			Object.defineProperty(document, 'scrollingElement', {
+				'get': scrollingElement
+			});
+		} else if ('__defineGetter__' in document) {
+			// Firefox < 4, Safari < 5, Opera < 12
+			document.__defineGetter__('scrollingElement', scrollingElement);
+		} else {
+			// fallback
+			document.scrollingElement = scrollingElement();
+			if ('attachEvent' in document) {
```

Comment:
Hmm, since this is only for IE ≤ 7 anyway (or are there other browsers that end up here?), we could move the `if` to right after the `else` to save one level of indentation.